### DISCUSSION TITLE
Is there a way to render html file automatically (like using a command) instead of clicking the icon?

### DISCUSSION THREAD

**Discussion by chunjiw** [fonsp/Pluto.jl#1647] [Questions about using Pluto] [answered]
I use Julia and Pluto remotely. To present my result to colleagues, I download the html file and upload it to the server manually, (and host it using GitHub page,) which is kind of annoying. Is there a way to automatically save the html file in the same machine of the Pluto notebook file?

**Comment by fonsp** [comment, marked as answer]
You might find what you need in https://github.com/JuliaPluto/PlutoSliderServer.jl#1-static-html-export or https://github.com/JuliaPluto/static-export-template

**Reply by chunjiw** [reply]
Thank you! It's just what I need.
